build.cmd

```diff
@@ -73,7 +75,7 @@ findstr /ir /c:".*Warning(s)" /c:".*Error(s)" /c:"Time Elapsed.*" "%_buildlog%"
 echo Build Exit Code = %BUILDERRORLEVEL%
 
 if "%outloop%" equ "true"  (
-	pushd setupfiles
+	pushd src\System.Private.ServiceModel\tools\setupfiles
```

Comment:
The same path is used twice in the one script. Would it be worth putting in a variable? I'm of two minds. If I want to run things by hand, I usually end up looking at the script and having a variable means my "in my head" cmd script parser needs to do more work, but it feels wrong to duplicate it.